employees.rb

```diff
@@ -0,0 +1,15 @@
+class Employee
+  attr_accessor :name, :department, :review, :satisfactory, :salary, :give_raise
+
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @department = nil
```

Comment:
You don't need to set these to nil, as all instance variables are nil by default if they're not set.